docker-compose/docker-compose.yaml

```diff
@@ -31,6 +31,11 @@ services:
     image: quay.io/mariadb-foundation/bb-master:master-web
     restart: unless-stopped
     container_name: master-web
+    environment:
+      - TITLE=MariaDB CI
+      - TITLE_URL=https://github.com/MariaDB/server
+      - BUILDMASTER_URL=https://buildbot.dev.mariadb.org/
+      - MQ_ROUTER_URL=ws://crossbar:8085/ws
```

Comment:
This seems wrong to me, you should rely on the env file and just define which environment variable to use:
```console
    environment:
      - TITLE
      - TITLE_URL
      - BUILDMASTER_URL
      - MQ_ROUTER_URL
```

And then the stack is started with either:
```console
docker-compose --env-file dev.env up -d
```
or 
```console
docker-compose --env-file prod.env up -d
```

See: https://gitlab.com/mariadb/downloads-mariadb-python3/-/blob/master/.gitlab-ci/deploy-dev.yml and https://gitlab.com/mariadb/downloads-mariadb-python3/-/tree/master/docker?ref_type=heads for examples.